### PR TITLE
Vinh/feature/ui/login

### PR DESCRIPTION
**Pull request by ThanhVinhLu**

### CODE REVIEW

File: src/pages/authentication/login/LoginForm.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ import RaisedButton from 'material-ui/RaisedButton';
 import './formStyle.css';
 
 const EMAIL_REGEX = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+const passwordRegex = new RegExp(
```
Comment (pvtri96) [inline src/pages/authentication/login/LoginForm.js]: This line should be `PASSWORD_REGEX = /.../i`

File: src/pages/authentication/register/RegisterForm.js [hunk 2]
```diff
@@ -6,7 +6,9 @@ import RaisedButton from 'material-ui/RaisedButton';
 import './RegisterForm.css';
 
 const EMAIL_REGEX = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
```
Comment (pvtri96) [inline src/pages/authentication/register/RegisterForm.js]: Please, stop hardcode import REGEX everywhere.

File: src/pages/conference/people/staff/detail/userInfo/tab.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Tabs, Tab } from 'material-ui/Tabs';
 // From https://github.com/oliviertassinari/react-swipeable-views
 import SwipeableViews from 'react-swipeable-views';
-import PersonalInfo from '../personal-info/personalInfo';
+import PersonalInfo from '../personalInfo/personalInfo';
```
Comment (pvtri96) [inline src/pages/conference/people/staff/detail/userInfo/tab.js]: This line should be `import ... from '../personalInfo'`;
Please use index.js instead.

**Comment by pvtri96** [comment]
@ThanhVinhLu Please, naming your Pull Request is VERY IMPORTANT.

File: src/pages/conference/people/staff/detail/userInfo/tab.js (hunk 3 above)
Comment (ThanhVinhLu) [reply src/pages/conference/people/staff/detail/userInfo/tab.js]: I will create a file for all regrex but there are some problem that i will discuss with you about PersonalInfo tomorow

Comment (pvtri96) [reply src/pages/conference/people/staff/detail/userInfo/tab.js]: Sure, I am very appreciated for that.

**Pull request merged**